tests/kafkatest/version.py

```diff
@@ -62,9 +62,6 @@ def _cmp(self, other):
 
         return LooseVersion._cmp(self, other)
 
-    def acl_command_supports_bootstrap_server(self):
-        return self >= V_2_1_0
```

Comment:
We never test below 2.1.0, so we don't need this check anymore.

Reply:
It seems other version checks have a similar issue, such as `reassign_partitions_command_supports_bootstrap_server`. Do you plan to remove them in this PR? Or should I file a follow-up to do more cleanup?